test/e2e/providers/kind.go

```diff
@@ -47,15 +53,7 @@ func (provider *KindProvider) RunCommandOnNodeExt(nodeName, cmd string, envs map
 }
 
 func (provider *KindProvider) GetKubeconfigPath() (string, error) {
-	homeDir, err := os.UserHomeDir()
-	if err != nil {
-		return "", fmt.Errorf("error when retrieving user home directory: %v", err)
-	}
-	kubeconfigPath := path.Join(homeDir, ".kube", "config")
-	if _, err := os.Stat(kubeconfigPath); os.IsNotExist(err) {
-		return "", fmt.Errorf("Kubeconfig file not found at expected location '%s'", kubeconfigPath)
-	}
-	return kubeconfigPath, nil
+	return *kindKubeconfig, nil
```

Comment:
Can you explain more about the code change in GetKubeconfigPath()?

Reply:
The kind provider of the test use hard code kubeconfig path. It always looks for ~/.kube/config. It's better to add a test argument to allow specifying kubeconfig path, just like remote provider.